### ISSUE TITLE
[BUG] Display doesn't show on  Computer screen after conection

### ISSUE THREAD

**Issue by mariojuggernaut22** [srevinsaju/guiscrcpy#306]
Essentially every time I try to connect my android device via guiscrcpy, the  dpad controls pop up but  the screen (as well as the  control bar for the screen) doesn't show up.. I tried with `adb.exe`, `scrcpy-server-v1.24`, `scrcpy.exe`

![image](https://user-images.githubusercontent.com/38630802/165947204-266636b8-676b-4744-94b1-d901669c4b7c.png) Setup Screen

![image](https://user-images.githubusercontent.com/38630802/165948003-66a8c08c-c4c5-4eb9-b141-39c4f97552ae.png)
Active screen

![image](https://user-images.githubusercontent.com/38630802/165948141-7aecc523-cfdf-4a4c-9a44-de7e3937bcfd.png)
Right after it starts up

![image](https://user-images.githubusercontent.com/38630802/165948194-1369f188-e90d-4633-bca8-dfbffccdd1d0.png)
 Shortly after that pic

**Comment by nightadmin** [comment]
+1
may be a bug in Android 12?

**Comment by nightadmin** [comment]
I found a way to fix it - choose your working drivers from the list (scrcpy --help) and push it into additional flags in the gui. Example:
![image](https://user-images.githubusercontent.com/78644136/169887610-c4c439be-8faa-4d8c-bc23-3b19b52f2b53.png)
It works fine for me.

**Comment by History-exe** [comment]
Android 10 also has no display

**Comment by srevinsaju** [comment]
This is actually not supposed to be a bug on guiscrcpy. Some issues on https://github.com/Genymobile/scrcpy might help to debug issues on screen display

**Comment by nightadmin** [comment]
Yes, so I get ideas for my solution from that repository. I think you can close this issue.

**Comment by darsparx** [comment]
So I'm having the same issue, I tried the driver list with quite a few of them none of them make the actual screen come up. Using scrcpy by itself works in the snap form but guiscrcpy from the same place in combination doesn't pull up the actual phone screen. This is on a samsung galaxy a53 as well

**Comment by srevinsaju** [comment]
@darsparx do you mean the guiscrcpy snap doesn't work, while scrcpy snap works? Can you also try out the guiscrcpy AppImage?

**Comment by darsparx** [comment]
Yeah with the snap it pulls up the toolbars and that's it. No screen. I can control it using the buttons on the toolbar but that's it. Flatpak wouldn't even see it. I'm also not sure what in flatseal I'd have to change for that but version has since been uninstalled. Tried to install with pip but couldn't figure out how to get all the requirements. Appimage same deal as the snap but this time the buttons for the bottom bar just disappear to the bottom right or back to center when I try to mess with them at all but still no image as far as I can tell anywhere on screen.

**Comment by History-exe** [comment]
> @darsparx do you mean the guiscrcpy snap doesn't work, while scrcpy snap works? Can you also try out the guiscrcpy AppImage?

I try the gui scrcpy Appimage, it works with my Android 10 device, but doesn't work on my Android 12 device.

**Comment by History-exe** [comment]
> > @darsparx do you mean the guiscrcpy snap doesn't work, while scrcpy snap works? Can you also try out the guiscrcpy AppImage?
> 
> I try the gui scrcpy Appimage, it works with my Android 10 device, but doesn't work on my Android 12 device.
> 
> And the Panels always goes to the down right corner of my computer screen, I can't move them.

Problem solved! In my case. The guiscrcpy Appimage version works on Android 10 but doesn't work on Android 12. This is because the scrcpy version too old. You need use the latest scrcpy.

**Comment by durgesh-tanuku** [comment]
If latest guiscrcpy version (v4.12.0) resolves this issue, how to install it on ubuntu 20.04 ?
snap is installing v4.9.0
